stories/LinterMessage.stories.tsx

```diff
@@ -153,20 +226,74 @@ storiesOf('LinterMessage', module).addWithChapters('all variants', {
                 simple add-ons, but are generally accepted.`,
                   ],
                 })}
-              />
-              <LinterMessage
-                inline
-                message={createMessage({
-                  type: 'error',
-                  message: 'Markup parsing error',
-                  description: [
-                    'There was an error parsing the markup document.',
-                    'malformed start tag, at line 1, column 26',
-                  ],
-                })}
-              />
-            </React.Fragment>
-          ),
+              />,
+            );
+          },
+        },
+        {
+          title: 'Two inline messages stacked',
+          sectionFn: () => {
+            return render(
+              <React.Fragment>
+                <LinterMessage
+                  inline
+                  message={createMessage({
+                    type: 'notice',
+                    message: 'Known JS library detected',
+                    description: [
+                      `JavaScript libraries are discouraged for
+                simple add-ons, but are generally accepted.`,
+                    ],
+                  })}
+                />
+                <LinterMessage
+                  inline
+                  message={createMessage({
+                    type: 'error',
+                    message: 'Markup parsing error',
+                    description: [
+                      'There was an error parsing the markup document.',
+                      'malformed start tag, at line 1, column 26',
+                    ],
+                  })}
+                />
+              </React.Fragment>,
+            );
+          },
+        },
+        {
+          title: 'Two inline messages stacked with one highlighted',
```

Comment:
It would be nice to add one more story to show a highlighted inline message, one with a triangle pointer at the top. The reason is because in the previous version with a 2px border, the triangle border wasn't affected so there are some specificity issues there. We'll want to make sure the border color of the triangle gets adjusted (it does).